src/components/interfaces/HMI_API.xml

```diff
@@ -2140,21 +2140,6 @@
     <function name="OnUpdateDeviceList" messagetype="notification">
       <description>Notification from HMI to SDL sent when HMI requires update of device list (i.e. when user clicks 'Change Device' button)</description>
     </function>
-    <function name="OnPhoneCall" messagetype="notification">
```

Comment:
@LuxoftAKutsan Yes, according [APPLINK-18854](https://adc.luxoft.com/jira/browse/APPLINK-18854) these functions were replaced by OnEventChanged (..), but were not removed